lib/util.js

```diff
@@ -11,19 +11,20 @@ var Debug;
 
 const formatRegExp = /%[sdj%]/g;
 exports.format = function(f) {
+  const args = arguments;
+  const len  = args.length;
+
   if (typeof f !== 'string') {
-    var objects = [];
-    for (var i = 0; i < arguments.length; i++) {
-      objects.push(inspect(arguments[i]));
+    let objects = new Array(len);
```

Comment:
That is not a good idea.
1. Semantics: creating a memory reference (variable or constant) that will for sure be modified should be declared as a variable, not a constant.
2. Scope: by using **let** the variable *objects* only exists in the scope of the **if** statement, but that is not ```true``` when using **const**